workflows/raredisease.nf

```diff
@@ -237,6 +243,16 @@ workflow RAREDISEASE {
             ch_score_config_sv
         )
         ch_versions = ch_versions.mix(RANK_VARIANTS_SV.out.versions)
+
+        VCFPARSER_SV (
+            RANK_VARIANTS_SV.out.vcf,
+            ch_variant_consequences
+        )
+
+        FILTER_VEP_SV(
+            VCFPARSER_SV.out.vcf,
+            ch_vep_filters
+        )
```

Comment:
I think we need to shift the order of these. vcfparser will add the most_sever_consequence tag which can contain effects on genes that are not part of the gene list.